packages/ui-components/src/chat/chat.tsx

```diff
@@ -133,6 +133,7 @@ function scrollToBottom(node: HTMLElement | null): void {
  */
 function Chat(props: ChatProps): JSX.Element {
     const [reply, setReply] = React.useState('');
+    const [isDisabled, setIsDisabled] = React.useState(true);
```

Comment:
Does this need to be a state at all? It look like it's  just computed state based on the reply text, e.g.

```suggestion
    const [reply, setReply] = React.useState('');
    const isDisabled = reply.trim().length === 0;
```

Relevant docs https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state and https://react.dev/learn/choosing-the-state-structure#avoid-redundant-state 
> When something can be calculated from the existing props or state, [don’t put it in state.](https://react.dev/learn/choosing-the-state-structure#avoid-redundant-state) Instead, calculate it during rendering.